articles/machines/resources-estimator.md

```diff
@@ -0,0 +1,149 @@
+---
+# Mandatory fields. See more on aka.ms/skyeye/meta.
+title: Quantum Development Kit Resources Estimator | Microsoft Docs 
+description: Overview of Microsoft's Quantum Development Kit Resources estimator 
+author: anpaz-msft
+ms.author: anpaz@microsoft.com 
+ms.date: 1/22/2019
+ms.topic: article
+uid: microsoft.quantum.machines.resources-estimator
+---
+
+# The ResourcesEstimator Target Machine
+
+As the name implies, the `ResourcesEstimator` estimates the resources 
+required to run a given instance of a Q# operation on a quantum computer.
+It accomplishes this by executing the quantum operation without actually 
+simulating the state of a quantum computer; for this reason, 
+it can estimate resources for Q# operations that use thousands of qubits.
+
+## Usage
+
+The `ResourcesEstimator` is just another type of target machine thus 
+it can be used to run any Q# operation. 
+
+As other target machines, to use it on a C# host program create an instance and pass it
+as the first parameter of the operation's `Run` method:
+
+```csharp
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+namespace Quantum.MyProgram
+{
+    class Driver
+    {
+        static void Main(string[] args)
+        {
+            ResourcesEstimator estimator = new ResourcesEstimator();
+            var res = MyQuantumProgram.Run(estimator).Result;
+            Console.WriteLine(estimator.ToTSV());
+        }
+    }
+}
+```
+
+As the example shows, the `ResourcesEstimator` provides a `ToTSV()` method to generate
+a table with tab-seperated-values (TSV) that can be saved into a file
+or written to the console for analysis. The output of the above program should look something like this:
+
+```Output
+Metric          Sum
+CNOT            1000
+QubitClifford   1000
+R               0
+Measure         4002
+T               0
+Depth           0
+Width           2
+BorrowedWidth   0
+```
+
+
+## Programmatically Retrieving the Estimated Data
+
+On top of a TSV table, the resources estimated can be retrieved programmatically
```

Comment:
Should it be "in addition to a TSV table" or "as an alternative to a TSV table"?